src/plugins/condition/criterion/TelemetryCriterion.js

```diff
@@ -116,34 +119,35 @@ export default class TelemetryCriterion extends EventEmitter {
         return this.telemetryObject && this.metadata && this.operation;
     }
 
-    /**
-     *  Subscribes to the telemetry object and returns an unsubscribe function
-     *  If the telemetry is not valid, returns nothing
-     */
-    subscribe() {
-        if (this.isValid()) {
-            this.unsubscribe();
-            this.subscription = this.telemetryAPI.subscribe(this.telemetryObject, (datum) => {
-                this.handleSubscription(datum);
-            });
-        } else {
-            this.handleSubscription();
-        }
-    }
+    requestLAD(options) {
+        options = Object.assign({},
+            options,
+            {
+                strategy: 'latest',
+                size: 1
+            }
+        );
 
-    /**
-     *  Calls an unsubscribe function returned by subscribe() and deletes any initialized data
-     */
-    unsubscribe() {
-        //unsubscribe from telemetry source
-        if (typeof this.subscription === 'function') {
-            this.subscription();
-        }
-        delete this.subscription;
+        return this.objectAPI.get(this.telemetryObjectIdAsString)
```

Comment:
We already have the telemetry object as `this.telemetryObject`. Can you reuse that instead of making a new call get the telemetry object?

Reply:
Sure. This was done with the intent of removing this.telemetryObject. If that is not needed we can reuse here.

Reply:
Actually. For now at least I'll leave this one as is. Even if this is being called twice I need to wait for this async call before requesting LAD. If I use this.telemetryObject as is it will not be loaded in time.